kuma/wiki/tests/test_templates.py

```diff
@@ -680,6 +680,7 @@ def test_tag_list(self):
         doc = pq(response.content)
         eq_(1, len(doc('#document-list ul.document-list li')))
 
+    @pytest.mark.xfail(reason='Requires migrations')
```

Comment:
I had to test this myself.  The test passes if you call tests normally, like ``py.test kuma/wiki/tests/test_templates.py::DocumentListTests``, but fails with ``--nomigrations``, with the error:

```
Traceback (most recent call last):
  File "/app/kuma/wiki/tests/test_templates.py", line 693, in test_tag_list_duplicates
    fr_tag.save()
  File "/usr/local/lib/python2.7/dist-packages/taggit/models.py", line 89, in save
    return super(TagBase, self).save(*args, **kwargs)
.. OMIT SOME LINES ... 
  File "/usr/local/lib/python2.7/dist-packages/MySQLdb/connections.py", line 36, in defaulterrorhandler
    raise errorvalue
IntegrityError: (1062, "Duplicate entry 'CSS R\xc3\xa9f\xc3\xa9rence' for key 'name'")
```
This means that "CSS Reference" is counted as different than "CSS Référence" with migrations, but as the same string without migrations.  This means the custom collation is only being applied when migrations are skipped and the tables are recreated directly. This happens both in Docker and Vagrant.

The test mentions http://bugzil.la/871638, an ISE on the wiki tag list page, fixed before the custom collation.  I'm more interested in [bug 1271552](https://bugzilla.mozilla.org/show_bug.cgi?id=1271552), where duplicate tags appear spontaneously, and [bug 776048](https://bugzilla.mozilla.org/show_bug.cgi?id=776048), where the custom collation was added.

As for marking it ``xfail``, it may be possible to make it conditionally skipped, with ``@pytest.mark.skipif(config.getvalue('migrations'))``, but I don't know what the config actually is, and ``xfail`` is fine with just 4 of them, but I'm a broken person and had to mention it.